docs/contributing/guides/style-guides/rust.md

```diff
@@ -38,6 +38,8 @@ Husky will automatically run the following when you attempt to commit changes:
 - `cargo clippy -- -D warnings`: A collection of lints to catch common mistakes
   and improve your Rust code. [Source
   Code](https://github.com/rust-lang/rust-clippy)
+- `taplo fmt`: Format TOML files to Arrow's style guide
+- `cspell`: Spell check all files in the repository
 
```

Comment:
We also have the markdown link check that runs during pre-commit.
The exact make targets that run during the test are:

- rust-test-all (combined target: rust-build rust-check rust-test rust-clippy rust-fmt)
- cspell-test
- toml-test
- python-test
- md-test-links
- editorconfig-test

Reply:
Yeah, I think it makes sense to list them all since all these checks will be running for the Rust projects